the-blue-alliance-ios/Core Data/EventRanking.swift

```diff
@@ -30,7 +30,34 @@ extension EventRanking: Managed {
         }
     }
     
-    static func insert(with model: TBAEventRanking, for event: Event, for team: Team, in context: NSManagedObjectContext) -> EventRanking {
+    var sortOrderInfoNames: [String] {
+        get {
+            return sortOrderInfoNamesArray as? Array<String> ?? []
+        }
+        set {
+            sortOrderInfoNamesArray = newValue as NSArray
+        }
+    }
+    
+    var infoString: String? {
+        get {
+            if !tieBreakers.isEmpty && !sortOrderInfoNames.isEmpty && tieBreakers.count == sortOrderInfoNames.count {
+                var string = ""
+                for (sortOrderName, sortOrderValue) in zip(sortOrderInfoNames, tieBreakers) {
```

Comment:
Maybe we try to make these two array names closer to one another so it’s more obvious they belong together? Like, maybe we call `sortOrderInfoNames` something like `tieBreakerNames` and then `tieBreakers` becomes `tieBreakerValues` ? Not sure - I haven’t thought about it much. I’ll leave that one up to you